includes/Traits/Google_Fonts.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Trait Google Fonts
+ *
+ * @package   Google\Web_Stories\Traits
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Traits;
+
+/**
+ * Trait Types
+ *
+ * @package Google\Web_Stories\Traits
+ */
+trait Google_Fonts {
+	/**
+	 * Register font in WordPress.
+	 *
+	 * @param string $font Name of font.
+	 */
+	public function register_google_font( $font ) {
+		wp_register_style(
+			$this->get_font_slug( $font ),
+			sprintf( 'https://fonts.googleapis.com/css2?family=%s:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap', $font ),
```

Comment:
We don't need all font variants on every screen. For example, on the plugin activation notice we probably only need `0,400` - loading everything is unnecessary and bloats things up.

Reply:
Part of the point this PR, to have a DNS lookup and uncached call. Even through these fonts are not needed at this second, 
1. They will be needed soon. 
2. By requesting an different url in the css, you are loading css and the overhead of another DNS look. That overhead, can be worse than downloading fonts you dont need at this time. Downloading font files is cheap, dns lookups and request for new css file.

Reply:
> They will be needed soon.

How is that so?